### ISSUE TITLE
Autostart malfunctioning on Fedora?

### ISSUE THREAD

**Issue by charleswilmot** [vajrasky/wallpapoz#22]
During startup, `xprop` fails, thus preventing the proper launch of `daemon_wallpapoz`

```
> journalctl

june 15 17:50:22 cx1g8 wallpapoz-autostart.desktop[2218]: daemon_wallpapoz: xprop -root _NET_CURRENT_DESKTOP 2>/dev/null failed at startup, retrying after 3 seconds...
june 15 17:54:22 cx1g8 wallpapoz-autostart.desktop[2218]: /usr/bin/daemon_wallpapoz: not lauching daemon_wallpapoz
```

I'm using fedora 32

**Comment by vajrasky** [comment]
Maybe you run on Wayland, instead of X. Try to check it.

https://ask.fedoraproject.org/t/xprop-issue-when-examining-terminal-window/7541

**Comment by charleswilmot** [comment]
I am indeed on Wayland!

Should I simply skip the `xprop` script at `/usr/bin/daemon_wallpapoz` and run `/usr/libexec/daemon_wallpapoz` at startup instaed?
______________________________
[Am I running Wayland?](https://unix.stackexchange.com/questions/202891/how-to-know-whether-wayland-or-x11-is-being-used)

**Comment by vajrasky** [comment]
I don't think that helps. I need to find a xprop replacement in Wayland. I need to find a command to change the desktop background in Wayland. Give me some time. I'll try to look a way to do it.
